common/lib/xmodule/xmodule/modulestore/tests/test_mongo_call_count.py

```diff
@@ -109,52 +150,43 @@ class CountMongoCallsCourseTraversal(TestCase):
         # (if you'll eventually access all the fields and load all the definitions anyway).
         (MIXED_SPLIT_MODULESTORE_BUILDER, None, False, True, 4),
         (MIXED_SPLIT_MODULESTORE_BUILDER, None, True, True, 38),
-        (MIXED_SPLIT_MODULESTORE_BUILDER, 0, False, True, 131),
+        (MIXED_SPLIT_MODULESTORE_BUILDER, 0, False, True, 38),
         (MIXED_SPLIT_MODULESTORE_BUILDER, 0, True, True, 38),
         (MIXED_SPLIT_MODULESTORE_BUILDER, None, False, False, 4),
         (MIXED_SPLIT_MODULESTORE_BUILDER, None, True, False, 4),
-        # TODO: The call count below seems like a bug - should be 4?
-        # Seems to be related to using self.lazy in CachingDescriptorSystem.get_module_data().
-        (MIXED_SPLIT_MODULESTORE_BUILDER, 0, False, False, 131),
```

Comment:
**Note to reviewers:** This `TODO` was addressed by passing a `request_cache` object to the modulestore in these tests.